common/concurrency/BUILD

```diff
@@ -26,6 +26,7 @@ cc_library(
         "//common/enforce_no_timer",
         "//common/exception",
         "//common/os",
+        "@com_google_absl//absl/synchronization",
```

Comment:
I don't know whether it's worth it to also remove the absl/synch dep from `common/BUILD` (or even whether it's possible).

Reply:
I'll give it a try!